parlai/core/torch_agent.py

```diff
@@ -279,21 +333,21 @@ def batchify(self, obs_batch, sort=False,
         """
         if len(obs_batch) == 0:
             return Batch(None, None, None, None, None, None, None, None, None,
-                         None)
+                         None, None)
```

Comment:
:/ this is getting crazy.

Can I recommend, just below the declaration of batch:
`Batch.__new__.__defaults__ = (None,) * len(Batch._fields)`

This makes it so everything has a default value of `None`, and we can just call `Batch()` here.

Reply:
c.f. https://stackoverflow.com/a/18348004